README.md

```diff
@@ -128,6 +128,13 @@ We would endlessly like to thank the following contributors
             <sub><b>8BitsLives .❤️</b></sub>
         </a>
     </td>
+    <td align="center">
+        <a href="https://github.com/keithchad">
+            <img src="https://avatars.githubusercontent.com/u/63049827?v=4" width="100;" alt="keithchad"/>
```

Comment:
This should not be changed since they are added automatically by a bot.

Reply:
> This should not be changed since they are added automatically by a bot.

I didn't change it personally, I found it was automatically changed

Reply:
@chepsi what might have caused this?

Reply:
CI Automatically creates this change. Don't bother about it.

Reply:
> CI Automatically creates this change. Don't bother about it.

Okay. Is there any other change I need to make?